bk-config/bin/create_registration_cert.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+
+
+if [ $# -eq 0 ]
+  then
+    echo "Usage: create_registration_cert.sh MINUTES"
+    echo ""
```

Comment:
i would use the same language as in `create_client_files.sh` for what these parameters mean

MINUTES: the length of time (in minutes) the certificate is valid for

Reply:
done